### PR TITLE
Added MRU_Open_File_Reuse_Current_Empty_Tab option

### PR DESCRIPTION
**Pull request by anishsane**
By default, while opening a file in a tab, if the current tab is
completely empty, it tries to find a tab which contains the required
file. If it does not find it, it creates a new tab for it.
However, if the current tab is completely empty, we can re-use it.

I sometimes just open vim and then open the files from the MRU menu. Initially, vim has only one tab page with no file. We can re-use that tab instead of creating a new one.

### CODE REVIEW

**Comment by anishsane** [comment]
Hi,
Any plans for considering this PR? Accept/Reject/Improve+accept?

File: plugin/mru.vim [hunk 1]
```diff
@@ -106,6 +106,14 @@ if !exists('MRU_Open_File_Use_Tabs')
     let MRU_Open_File_Use_Tabs = 0
 endif
 
+" When opening a file from the MRU list in a tab, if the current tab is empty,
+" do not create a new tab. With the below variable set to 1, it will use the
+" current empty tab.
+
+if !exists('MRU_Open_File_Reuse_Current_Empty_Tab')
```
Comment (yegappan) [inline plugin/mru.vim:113]: We can make this as the default behavior. Can you remove this new setting?

File: plugin/mru.vim [hunk 2]
```diff
@@ -314,9 +322,14 @@ function! s:MRU_Open_File_In_Tab(fname, esc_fname)
 	    " Goto the tab containing the file
 	    exe 'tabnext ' . i
 	else
-	    " Open a new tab as the last tab page
-	    tablast
-	    exe 'tabnew ' . a:esc_fname
+		if (winnr("$") == 1) && (bufname("") == "") && !&l:modified && g:MRU_Open_File_Reuse_Current_Empty_Tab
```
Comment (yegappan) [inline plugin/mru.vim:325]: Can you remove the check for g:MRU_Open_File_Reuse_Current_Empty_Tab and make this the default behavior?

The l: prefix is not needed for modified. Can you remove the l: prefix?

Comment (anishsane) [reply plugin/mru.vim:325]: Are you suggesting that we can get rid of the `MRU_Open_File_Reuse_Current_Empty_Tab` option altogether and make it as default (as if MRU_Open_File_Reuse_Current_Empty_Tab was set to 1 in my current code)?
I would like that.
Let me know. I will update my code and the PR.

Comment (yegappan) [reply plugin/mru.vim:325]: Yes. We don't need the MRU_Open_File_Reuse_Current_Empty_Tab option. We can make this behavior the default behavior.

**Comment by anishsane** [comment]
Sent a separate pull request. Closing this one.

**Pull request closed without merging**